LANGUAGES/languages.json

```diff
@@ -370,16 +370,16 @@
         "STR_WEB_TOKEN_MISMATCH":"Daemon status updated, please retry later.",
         "STR_WEB_SERVICE_BUSY":"Service is busy, please retry later.",
         "STR_MENU_VTSI_CREATE":"Generate VTSI File",
-        "STR_VTSI_CREATE_TIP":"This time will not write to the device, but only generate a VTSI file#@Continue?",
-        "STR_VTSI_CREATE_SUCCESS":"VTSI file created successfully!#@You can use Rufus(3.15+) to write it to the device so as to complete the installation of Ventoy.",
-        "STR_VTSI_CREATE_FAILED":"VTSI file created failed.",
+        "STR_VTSI_CREATE_TIP":"This will not write to the device, only generating a VTSI file#@Continue?",
```

Comment:
This would be better written as:

_A VTSI file will be generated. This will not write to the device._
or
_This will not write to the device. Only a VSTI file is being generated._

The gerund in the second phrase reads awkwardly.

Reply:
going to fix this hang on